src/server/person/personroutes.js

```diff
@@ -3,14 +3,15 @@
 const personLookup = require('./personlookup');
 const personMapping = require('./personmapping');
 const router = require('express').Router();
+const thirtyfourHoursInMilliseconds = 122400;
```

Comment:
1. Kanskje det er bedre å sette inn utregningen `34*60*60*1000`, da slipper man å tvile på om tallet man ser er riktig når man leser koden.
2. Ved å ha _verdien_ av const-en i navnet er det risiko for at den blir utdatert, noe som kanskje allerede har skjedd her, siden det ser ut til å være antall sekunder den representerer nå? Greit å ha informasjon kun _ett_ sted - koster ikke så mye å lete opp verdien av en variabel.